tests/validators.py

```diff
@@ -278,7 +281,7 @@ def validate_forward():
             resp = requests.get("http://localhost:5123")
             if resp.status_code == 200:
                 break
-        except:
+        except requests.RequestException:
```

Comment:
Couldn't there be a ConnectionError https://requests.readthedocs.io/en/master/_modules/requests/exceptions/ ?

Reply:
`ConnectionError` inherits from `RequestException`, so catching `RequestException` should catch that as well, along with all the other exceptions in that page.